### ISSUE TITLE
doesn't work

### ISSUE THREAD

**Issue by retyui** [mrousavy/clean-rn#3]
```
npx clean-rn     

Need to install the following packages:
  clean-rn@1.0.2
Ok to proceed? (y) 
clean-rn: Cleaning React Native caches for /Users/i/.npm/_npx/f11239abab140955..
node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module '/Users/i/.npm/_npx/f11239abab140955/node_modules/shelljs/src/exec-child.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.17.1
node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module '/Users/i/.npm/_npx/f11239abab140955/node_modules/shelljs/src/exec-child.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.17.1
node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module '/Users/i/.npm/_npx/f11239abab140955/node_modules/shelljs/src/exec-child.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.17.1

```

**Comment by ahmedrowaihi** [comment]
My [windows support] PR solves this, I removed the libraries and hardcoded paths, introduced nodejs built-in methods, and hopefully, @mrousa merges it

https://github.com/mrousavy/clean-rn/pull/2

**Comment by ahmedrowaihi** [comment]
Hey @retyui 
Marc has merged the PR..
try again and kindly close the issue if all went well..

**Comment by retyui** [comment]
I will close it, when a new verion will be released

**Comment by retyui** [comment]
@ahmedrowaihi your fix doesn't work on macos

```java
npx github:mrousavy/clean-rn      
                                                                                                             
Need to install the following packages:
  github:mrousavy/clean-rn
Ok to proceed? (y) 
Could not parse package.json. make sure your terminal is at the root of your React Native project
This is not a React Native project

ls -la package.json                                                                                                                
-rw-r--r--  1 myUser  staff  752 Jul 31 18:33 package.json
```

**Comment by ahmedrowaihi** [comment]
Update: The package is updated with fixes, you may try using it now..
1.0.3

**Comment by retyui** [comment]
fixed in 1.0.3
